src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -49,48 +49,61 @@
   public static final String PYTHON = "python";
   public static final String RUBY = "ruby";
 
-  private static final String DEFAULT_SNIPPET_FILE = "discovery_fragment.snip";
+  private static final String ADC_SNIPPET_FILE = "discovery_fragment.snip";
```

Comment:
would it be easier to read to have instead "discovery_fragment_3lo", etc. (the specifics being a suffix rather than prefix)?